### PR TITLE
Enable more prefix styling options

### PR DESCRIPTION
**Pull request by OlsonDev**
Add command line options:
- names
- name-separator
- colors

Update lodash to 4.5.1 to use _.get(object, path)
- See https://github.com/lodash/lodash/wiki/Changelog
- _.pluck is now shorthanded _.map(objects, propertyName)
- _.contains was removed in favor of _.includes

Fix typo in test/test-functional.js

### CODE REVIEW

**Comment by OlsonDev** [comment]
This will close #27 and goes above and beyond by providing prefix-coloring. This is how I'm using it:

```
concurrently -p "[{index}:{name}]" -n "main,server,browser" -c "bgBlue.bold,bgMagenta.bold,bgGreen.bold" "npm run build-main" "npm run build-server" "npm run build-browser"
```

File: src/main.js [hunk 1]
```diff
@@ -21,9 +21,18 @@ var config = {
...
     prefix: 'index',
 
+    // List of custom names to be used in prefix template
+    names: '',
+
+    // What to split the list of custom names on
+    nameSeparator: ',',
+
+    // Comma-separated list of chalk color paths to use on prefixes.
+    colors: 'gray.dim',
```
Comment (Sicria) [inline src/main.js]: Could this be changed to ``nameColors`` to be a bit more descriptive?

**Comment by Sicria** [comment]
LGTM :+1: Thank you.

**Comment by OlsonDev** [comment]
I renamed it to `prefixColors` instead of `nameColors` (like you suggested) because the color applies to the entire prefix, not just the name.

**Comment by JakeGinnivan** [comment]
Any chance of this being merged/released? It would be quite handy.

**Comment by OlsonDev** [comment]
@JakeGinnivan If you want to use it while we wait for @kimmobrunfeldt, you can change your `package.json` to have `"devDependencies"` of `"concurrently": "OlsonDev/concurrently#master"`. That's what I've been doing, anyway.

**Comment by kimmobrunfeldt** [comment]
Sorry for taking so long to review this. Could you remove the version bump of concurrently in package.json? The bump is automated in my release process. Otherwise looks good.

**Comment by OlsonDev** [comment]
That work?

**Comment by kimmobrunfeldt** [comment]
Released as 2.1.0 https://github.com/kimmobrunfeldt/concurrently/releases/tag/2.1.0

**Pull request merged**